app/client/cli/keys.go

```diff
@@ -0,0 +1,465 @@
+				keybasePath, err := filepath.Abs(pocketDir + keybaseSuffix)
+				if err != nil {
+					return err
+				}
+				kb, err := keybase.NewKeybase(keybasePath)
+				if err != nil {
+					return err
+				}
+
+				addresses, _, err := kb.GetAll()
+				if err != nil {
+					return err
+				}
+
+				for _, addr := range addresses {
+					fmt.Println(addr)
+				}
+
+				return kb.Stop()
+			},
+		},
+		{
+			Use:     "Get <addrHex>",
+			Short:   "Get <addrHex>",
+			Long:    "Get the keypair <addrHex> from the keybase",
+			Aliases: []string{"get"},
+			Args:    cobra.ExactArgs(1),
+			RunE: func(cmd *cobra.Command, args []string) error {
+				// Unpack CLI args
+				addrHex := args[0]
+
+				// Open the debug keybase at the specified path
+				pocketDir := strings.TrimSuffix(dataDir, "/")
+				keybasePath, err := filepath.Abs(pocketDir + keybaseSuffix)
+				if err != nil {
+					return err
+				}
+				kb, err := keybase.NewKeybase(keybasePath)
+				if err != nil {
+					return err
+				}
+
+				kp, err := kb.Get(addrHex)
+				if err != nil {
+					return err
+				}
+
+				fmt.Printf("Address: %s\nPublic Key: %s\n", addrHex, kp.GetPublicKey().String())
+
+				return kb.Stop()
+			},
+		},
+	}
+	return cmds
+}
+
+func keysExportCommands() []*cobra.Command {
+	cmds := []*cobra.Command{
+		{
+			Use:     "Export <addrHex>",
+			Short:   "Export <addrHex>",
+			Long:    "Exports the private key of <addrHex> as a raw or JSON encoded string",
+			Aliases: []string{"export"},
+			Args:    cobra.ExactArgs(1),
+			RunE: func(cmd *cobra.Command, args []string) error {
+				// Unpack CLI args
+				addrHex := args[0]
+
+				// Open the debug keybase at the specified path
+				pocketDir := strings.TrimSuffix(dataDir, "/")
+				keybasePath, err := filepath.Abs(pocketDir + keybaseSuffix)
+				if err != nil {
+					return err
+				}
+				kb, err := keybase.NewKeybase(keybasePath)
+				if err != nil {
+					return err
+				}
+
+				pwd = readPassphrase(pwd)
+
+				// Determine correct way to export private key
+				var exportString string
+				exportAs = strings.ToLower(exportAs)
+				if exportAs == "json" {
+					exportString, err = kb.ExportPrivJSON(addrHex, pwd)
+					if err != nil {
+						return err
+					}
+				} else if exportAs == "raw" {
+					exportString, err = kb.ExportPrivString(addrHex, pwd)
+					if err != nil {
+						return err
+					}
+				} else {
+					return fmt.Errorf("invalid export format: got %s, want [raw]/[json]", exportAs)
+				}
+
+				if err := kb.Stop(); err != nil {
+					return err
+				}
+
+				// Write to stdout or file
+				if outputFile == "" {
+					fmt.Println(exportString)
+					return nil
+				}
+
+				return writeOutput(exportString, outputFile)
+			},
+		},
+	}
+	return cmds
+}
+
+func keysImportCommands() []*cobra.Command {
+	cmds := []*cobra.Command{
+		{
+			Use:     "Import <privateKeyString>",
+			Short:   "Import <privateKeyString>",
+			Long:    "Imports <privateKeyString> into the keybase",
```

Comment:
In `Long`, use `[]` to denote optional parameters and add some details around defaults.

Ditto elsewhere

Reply:
I have added the `[]` parts but when the user is able to see the long description is when they will see the command specific flags and cobra automatically attaches the default values if there are any to these at that point

![image](https://user-images.githubusercontent.com/53987565/219867020-676bfc14-1d99-49d6-8ce2-320afb3795b1.png)